autonomi/src/client/high_level/files/archive_public.rs

```diff
@@ -161,12 +161,6 @@ impl Client {
             .to_bytes()
             .map_err(|e| PutError::Serialization(format!("Failed to serialize archive: {e:?}")))?;
 
-        #[cfg(feature = "loud")]
-        println!(
-            "Uploading public archive referencing {} files",
-            archive.map().len()
-        );
-
```

Comment:
Although I agree it's not the prettiest way to do it, it's an excellent UX for CLI. Until we find another way I think we need to keep it